src/catalog/catalog.cpp

```diff
@@ -91,12 +95,13 @@ void Catalog::TearDown() {
 bool Catalog::CreateDatabase(const common::ManagedPointer<transaction::TransactionContext> txn, const std::string &name,
                              const bool bootstrap, const catalog::db_oid_t db_oid) {
   // Instantiate the DatabaseCatalog
-  DatabaseCatalog *dbc = postgres::Builder::CreateDatabaseCatalog(catalog_block_store_, db_oid);
+  DatabaseCatalog *dbc = postgres::Builder::CreateDatabaseCatalog(catalog_block_store_, db_oid, garbage_collector_);
   txn->RegisterAbortAction([=](transaction::DeferredActionManager *deferred_action_manager) {
     dbc->TearDown(txn);
     delete dbc;
   });
   bool success = Catalog::CreateDatabaseEntry(common::ManagedPointer(txn), db_oid, name, dbc);
+  // TODO(Matt): should this be if (success && bootstrap), or should we bootstrap regardless of db name uniqueness?
```

Comment:
Would this clobber the existing database? Right now it looks like it would probably trip some asserts.

Reply:
We discussed this in the deferred events meeting last week, so I can remove this TODO.